beps/pub-sub.rst

```diff
@@ -29,6 +29,8 @@ The pub/sub protocol is based on the DHT protocol [#BEP-5]_ with the following m
 
 The value of K is set to one.
 
+The routing table's home bucket and its sibling SHOULD have a capacity of at least 8 nodes each. When the home bucket is split all but one node from the former home bucket's sibling SHOULD be evicted from the routing table.
```

Comment:
The part about splitting is wrong. You don't know in which half of the split they will end up, so you might end up discarding anywhere from 0 to 7 nodes.